### PR TITLE
Add support for specifying certificate duration at runtime.

### PR DESCRIPTION
**Pull request by jcbsmpsn**

### CODE REVIEW

File: build/BUILD [hunk 1]
```diff
@@ -87,7 +87,7 @@ DOCKERIZED_BINARIES = {
 
 [docker_bundle(
     name = binary,
-    images = {"gcr.io/google_containers/%s:{STABLE_DOCKER_TAG}" % binary: binary + "-internal"},
+    images = {"gcr.io/google_containers/%s:v1.6.9" % binary: binary + "-internal"},
```
Comment (mikedanese) [inline build/BUILD]: revert?

**Comment by jcbsmpsn** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

**Comment by jcbsmpsn** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by mikedanese** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by jcbsmpsn** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by jcbsmpsn** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by jcbsmpsn** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

**Comment by mikedanese** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by jcbsmpsn** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by jcbsmpsn** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by jcbsmpsn** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Pull request merged**